app-doc/audacity/audacity-9999.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="HTML reference manual for Audacity"
+HOMEPAGE="https://www.audacityteam.org/"
+
+if [[ ${PV} = 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/audacity/audacity-manual.git"
+else
+	KEYWORDS="amd64 riscv"
```

Comment:
Don't commit as stable - ~amd64 and just drop riscv. We can rekeyword it later. Put it under SRC_URI too.

Reply:
Both changes done.